src/mqueryfront/src/QueryResultsStatus.js

```diff
@@ -206,6 +219,7 @@ class QueryResultsStatus extends Component {
                         <thead>
                             <tr>
                                 <th>Matches</th>
+                                <th style={{ width: "32%" }}>SHA256</th>
```

Comment:
weirdly specific

Use bootstrap classes (probably something like `col-md-4`). Bootstrap grid system will probably also allow to automatically hide/flow the columns on resize